src/main/scala/tech/cryptonomic/conseil/config/Security.scala

```diff
@@ -12,10 +50,12 @@ object Security {
 
     /**
       * Determines whether a given API key is valid.
-      * @param candidateApiKey  The given API key
-      * @return                 True is valid, false otherwise.
+      *
+      * @param candidateApiKey The given API key
+      * @return True is valid, false otherwise.
       */
-    def validateApiKey(candidateApiKey: String): Boolean = keys(candidateApiKey)
+    def validateApiKey(candidateApiKey: String): Future[Boolean] =
+      nautilusCloudKeys.get.map(ncKeys => (ncKeys ++ keys)(candidateApiKey)).unsafeToFuture()
```

Comment:
For un-named sets like in this case, would you say that using `contains` explicitly is easier to understand?

Reply:
ok, changed